content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -189,3 +189,21 @@ const userinfo = await prisma.userInfo.findMany({
 ```
 
 Currently, Prisma Client allows you to update a view if the underlying database allows it, without any additional validation.
+
+## Different types of views
```

Comment:
```suggestion
## Types of views
```

Reply:
With this edit I realize that the third kind is missing then: The normal ones, not updatable and not materialized.

Maybe:
```suggestion
## Special types of views
```